extensions/amp-viz-vega/0.1/amp-viz-vega.js

```diff
@@ -0,0 +1,166 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {isJsonScriptTag, childElementsByTag} from '../../../src/dom';
+import {isExperimentOn} from '../../../src/experiments';
+import {tryParseJson} from '../../../src/json';
+import {isLayoutSizeDefined} from '../../../src/layout';
+import {dev, user} from '../../../src/log';
+import {toggle} from '../../../src/style';
+
+/** @const */
+const EXPERIMENT = 'amp-viz-vega';
+
+export class AmpVizVega extends AMP.BaseElement {
+
+  /** @param {!AmpElement} element */
+  constructor(element) {
+    super(element);
+
+    /** @private {?Element} */
+    this.container_ = null;
+
+    /** @private {?JSONType} */
+    this._spec = null;
```

Comment:
muscle memory from Dart days :) Sadly, I am sure I will make this mistake again.